src/main/io/serial_cli.c

```diff
@@ -358,8 +363,8 @@ static const char * const lookupTableGPSSBASMode[] = {
 };
 #endif
 
-static const char * const lookupTableCurrentSensor[] = {
-    "NONE", "ADC", "VIRTUAL"
+static const char * const lookupTableCurrentMeter[] = {
```

Comment:
Alowing ADC1 - ADC4 as current meter input will work better when the values are consecutive ...